python/ray/tests/kuberay/test_autoscaling_e2e.py

```diff
@@ -33,26 +33,18 @@
 )
 
 logger = logging.getLogger(__name__)
-logging.basicConfig(
-    level=logging.INFO,
-    format="[%(levelname)s %(asctime)s] " "%(filename)s: %(lineno)d  " "%(message)s",
-)
 
 # This image will be used for both the Ray nodes and the autoscaler.
 # The CI should pass an image built from the test branch.
-RAY_IMAGE = os.environ.get("RAY_IMAGE", "rayproject/ray:448f52")
+RAY_IMAGE = os.environ.get("RAY_IMAGE", "rayproject/ray:nightly-py38")
```

Comment:
No particular reason for this default besides the fact that I use a py38 environment locally.

Reply:
(The CI specifies and image built from the PR branch.)